techdocs/docs/getting-started/quick-start-guide.md

```diff
@@ -24,9 +24,9 @@ root.render(
 );
 ```
 
-> ### `Note`
-> SSOProvider has optional props `backendURL`, `idpHint`, `onRefreshExpiry`, and `postLoginRedirectURL`.  
-> See TypeScript Type SSOProviderProps for more details on usage.
+!!! note "Note"
```

Comment:
```suggestion
!!! info "Note"
```
Can we try this to see which we like better?

Reply:
note preview:
![note_test](https://github.com/bcgov/citz-imb-sso-react/assets/78506153/993235ca-9024-483f-9516-3f3dcbf62930)
info preview:
![info_test](https://github.com/bcgov/citz-imb-sso-react/assets/78506153/618b701e-16c6-44be-ba92-0c3624241d00)